backend/config/settings.py

```diff
@@ -119,16 +123,60 @@
     "DEFAULT_PERMISSION_CLASSES": [
         "rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly",
     ],
+
     'DEFAULT_AUTHENTICATION_CLASSES': [
-        # token authentication
-        'rest_framework.authentication.TokenAuthentication',
+
+        # JWT authentication
+        'rest_framework_simplejwt.authentication.JWTAuthentication',
+
+        # # token authentication
+        # 'rest_framework.authentication.TokenAuthentication',
 
         # # session authentication
         # 'rest_framework.authentication.BasicAuthentication',
         # 'rest_framework.authentication.SessionAuthentication',
     ],
 }
 
+SIMPLE_JWT = {
+    "ACCESS_TOKEN_LIFETIME": timedelta(minutes=5),
```

Comment:
I think, 5 minutes lifetime is too short.

Reply:
Changed timedelta(day=1)